docker-compose.yml

```diff
@@ -26,15 +26,29 @@ services:
     pattern-library:
         image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
         restart: always
+    blog-articles_postgres:
+        image: postgres:11.2-alpine
+        volumes:
+            - data-blog-articles:/var/lib/postgresql/data
+        healthcheck:
+            test: ["CMD", "pg_isready", "--username", "postgres"]
+            interval: 5s
     blog-articles_fpm:
         image: ${DOCKER_NAMESPACE}/content-store:${REVISION_CONTENT_STORE}
-        volumes:
-            - ./.docker/blog-articles/data:/app/data/
+        command: >
+            sh -c '
+                while ! nc blog-articles_postgres 5432; do sleep 1; done
+                bin/console doctrine:migrations:migrate --no-interaction
+                php-fpm
+            '
```

Comment:
could eventually be put in the `CMD` of the image... but it would prevent using different persistence options if there will be any